docs/docs/contribute/environment-setup.mdx

```diff
@@ -0,0 +1,82 @@
+# Setup your Development Environment
+
+Setting up your local copy of the project for development and testing is essential. Follow the steps below to configure your environment.
+
+:::info "Install prerequisites"
+
+Before you start contributing to this project, you will need **Node.js@18** and **npm@8** installed.
```

Comment:
Minor thing but we usually drop `.js` when referring to a specific version on node.

```suggestion
Before you start contributing to this project, you will need **node@18** and **npm@8** installed.
```